api/python/provisioner/ui/error.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+#
+
+
+class PrvsnrException(Exception):
```

Comment:
we already have [a set of base exceptions](https://github.com/Seagate/cortx-prvsnr/blob/ddd9651ef721c2778a420f04624816b72695fb74/api/python/provisioner/errors.py#L50-L66). Please don't introduce a new one.

Reply:
addressed